pkg/apiserver/apiserver_test.go

```diff
@@ -159,159 +147,182 @@ func generateTestKey() (*rsa.PrivateKey, error) {
 	return apiKeyPair, nil
 }
 
-func TestGetSelfSignedCert(t *testing.T) {
-	signingKey, err := generateTestKey()
-	if err != nil {
-		t.Errorf("error generating keys: %v", err)
-	}
+func doGetRequest(url string) *httptest.ResponseRecorder {
+	app := &cdiAPIApp{}
+	app.composeUploadTokenAPI()
 
-	signingKeySecret, err := keystest.NewPrivateKeySecret("cdi", apiSigningKeySecretName, signingKey)
-	if err != nil {
-		t.Errorf("error creating secret: %v", err)
-	}
+	req, _ := http.NewRequest("GET", url, nil)
```

Comment:
Missing param is err, which we can easily check with Expect(err).ToNot(HaveOccurred())